src/pages/order-item/order-item.js

```diff
@@ -28,24 +28,49 @@ const OrderItem = ({ id }) => {
     selectedOrder: Orders.selectedOrder,
     orderLoading: Orders.orderLoading
   }));
+
+  useEffect(() => {
+    id && dispatch(getOrder(id));
+  }, [dispatch, id]);
+
   const submitStatus = ['CREATED', 'CONFIRMED'];
 
   const handleTabChange = (e, newValue) => {
     setTabValue(newValue);
   };
 
   const handleFormSubmit = (order) => {
+    const items = order.items.map((item) => ({
+      product: item?.product._id,
+      quantity: item.quantity,
+      isFromConstructor: !item.product._id,
+      options: {
+        size: item.options.size._id,
+        sidePocket: item.options.sidePocket
+      }
```

Comment:
create in utils file order (or use already created) and put this func into utils